modules/gke-cluster-autopilot/README.md

```diff
@@ -1,10 +1,23 @@
-# GKE cluster Autopilot  module
+# GKE Autopilot cluster module
 
-This module allows simplified creation and management of GKE Autopilot clusters. Some sensible defaults are set initially, in order to allow less verbose usage for most use cases.
+This module offers an opinionated way to create and manage Google Kubernetes Engine (GKE) [Autopilot clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview). With its sensible default settings based on best practices and authors' experience as Google Cloud practitioners, the module accommodates for many common use cases out-of-the-box, without having to rely on verbose configuration.
```

Comment:
nit: we try not to be opinionated in modules, especially compared to other GCP TF modules who tweak the cluster after creation :)

Reply:
Understood. I will rollback this change

Reply:
Documentation updated to avoid ambiguous/unwanted wording